geomesa-gs-plugin/geomesa-bigtable-gs-plugin/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>geomesa-gs-plugin</artifactId>
         <groupId>org.locationtech.geomesa</groupId>
-        <version>1.2.0-SNAPSHOT</version>
+        <version>1.2.7-SNAPSHOT</version>
```

Comment:
special only bc its in a parent tag...all the parent tag references are updated by maven release:prepare the reason it was off before is bc our build for releases doesn't use the bigtable profile